.gitignore

```
@@ -23,3 +23,8 @@ coverage
 *.njsproj
 *.sln
 *.sw?
+
+# Synpress directories and files
+.env
+tests
```

keeping `tests` out of SCM seems like a recipe for confusion. What is making those files? Can it be put somewhere else?

---

@turadg The end-to-end test cases reside in the `test` folder. 

When we run these tests, a folder is automatically generated to store screenshots and videos of the test execution. Specifically, it creates two directories:

```
Screenshots folder: tests/e2e/screenshots
Videos folder: tests/e2e/videos
```

`tests` folder is excluded from SCM to avoid cluttering the repo with these large/temp files.

---

I get that. But it can be put somewhere else and I think that it should be to avoid confusion. It's quite surprising that `test` is in SCM and `tests` is ignored.

https://docs.cypress.io/guides/references/configuration#Screenshots says the default directory is `cypress/screenshots`. Why is this different?

---

@turadg In `@agoric/synpress` repo, we follow a different folder structure for organizing our end-to-end test files. Specifically:
```
Screenshots folder: tests/e2e/screenshots
Videos folder: tests/e2e/videos
Tests folder: tests/e2e/spec
```
This structure follows the one used in the `synpress` repo, which we aimed to remain consistent with.
`inter-dapp` repo already has `test` folder. To have `tests` and `test` folder in the same repo, would have been confusing, so I went with adding end to end tests in `test/e2e/spec` directory.

---

i don't understand why Synpress changed the default paths that Cypress used, but I won't argue with it.

I do request that you ignore the directories that have large files, rather than everything that might appear in a `tests` directory. The confusion of gitignoring `tests` is already evident by virtue of this discussion.

```suggestion
screenshots
videos
```

---

@turadg Updated the `.gitignore`